### PR TITLE
Move sample using appengine images to this repository, add tests.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Currently here: https://github.com/GoogleCloudPlatform/appengine-images-guestbook-python
From here: https://cloud.google.com/appengine/docs/python/images/usingimages

### CODE REVIEW

File: appengine/images/tests/test_guestbook.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        # You can also pass headers, query arguments etc.
+        request = webapp2.Request.blank('/')
+        # Get a response for that request.
+        response = request.get_response(main.app)
+
+        # Let's check if the response is correct.
+        self.assertEqual(response.status_int, 200)
+
+    @mock.patch('appengine.images.main.images')
+    def test_post(self, mock_images):
```
Comment (tmatsuo) [inline appengine/images/tests/test_guestbook.py]: Can you check whether the resize() is called once?

File: appengine/images/tests/test_guestbook.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        request = webapp2.Request.blank('/img?img_id=123')
+        response = request.get_response(main.app)
+
+        self.assertEqual(response.status_int, 500)
+
+    @mock.patch('appengine.images.main.images')
+    def test_get(self, mock_images):
+        mock_images.resize.return_value = 'asdf'
+        request = webapp2.Request.blank(
+            '/sign',
```
Comment (tmatsuo) [inline appengine/images/tests/test_guestbook.py]: Isn't this test for Image.get()?

Comment (jeffmendoza) [inline appengine/images/tests/test_guestbook.py]: Trying to test a get on '/' after a '/sign'

File: appengine/images/tests/test_guestbook.py (hunk 1 above)
Comment (jeffmendoza) [inline appengine/images/tests/test_guestbook.py]: will do

**Comment by jeffmendoza** [comment]
@tmatsuo Updated with assert.

File: appengine/images/tests/test_guestbook.py (hunk 2 above)
Comment (tmatsuo) [inline appengine/images/tests/test_guestbook.py]: then test_post_and_get or something better name?

File: appengine/images/tests/test_guestbook.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            POST={'content': 'asdf'},
+            )
+        response = request.get_response(main.app)
+        mock_images.resize.assert_called_once()
+
+        # Correct response is a redirect
+        self.assertEqual(response.status_int, 302)
+
+    def test_img(self):
```
Comment (tmatsuo) [inline appengine/images/tests/test_guestbook.py]: Can you also test the success case?

File: appengine/images/tests/test_guestbook.py (hunk 2 above)
Comment (jeffmendoza) [inline appengine/images/tests/test_guestbook.py]: sounds good.

File: appengine/images/tests/test_guestbook.py (hunk 3 above)
Comment (jeffmendoza) [inline appengine/images/tests/test_guestbook.py]: I'm not sure how to generate a webapp2(webob) Request post object with multipart form data to include the image. Do you know?

Comment (tmatsuo) [inline appengine/images/tests/test_guestbook.py]: Maybe you can just add an entity in datastore and call /img?img_id=123
What do you think?

Comment (jeffmendoza) [inline appengine/images/tests/test_guestbook.py]: I'll try it out.

**Comment by jeffmendoza** [comment]
Get image positive test case is in now.

**Comment by tmatsuo** [comment]
LGTM

**Comment by jeffmendoza** [comment]
Squashed.

**Pull request merged**